edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/Constants.cs

```diff
@@ -15,7 +15,9 @@ public class Constants
         public const string SessionStorePartitionKey = "sessions";
 
         public const string InternalOriginInterface = "internal";
-        public const string DownstreamOriginInterface = "downstream";        
+        public const string DownstreamOriginInterface = "downstream";
```

Comment:
What's this used for?

Reply:
So, for all messages that pass through the system, we stamp the "downstream" or "internal" system property that you can use to filter upon.